upgrade/steps.go

```diff
@@ -312,7 +312,7 @@ func InformGitHub(
 
 	reviewerFlag := ""
 	if ctx.PrReviewers != "" {
-		reviewerFlag = fmt.Sprintf("--reviewer=%s", ctx.PrReviewers)
+		reviewerFlag = fmt.Sprintf("--reviewer=\"%s\"", ctx.PrReviewers)
```

Comment:
This will not fix the issue. The issue is that `reviewerFlag` gets literally quoted into the command as
`gh pr create --assignee @me --base main --head test "" --title hiwhatsup --body loremipsum`.

Reply:
this works:
`gh pr create --assignee @me --base main --head test --reviewer "" --title hiwhatsup --body loremipsum` so we can just get rid of the entire if statement here!